python/py-pgspecial/Portfile

```diff
@@ -15,20 +15,18 @@ description         Meta-commands handler for Postgres Database
 long_description    This package provides an API to execute meta-commands \
                     (AKA “special”, or “backslash commands”) on PostgreSQL.
 
-python.versions     27 36 37 38 39
+python.versions     36 37 38 39
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 
-checksums           rmd160  4fc93087c497624f1e0dd7798f8d172f495c3ab4 \
-                    sha256  f65c74a7ecfd4d6af3feb963a1bf8a612e5882731f69afd06ae66ffee13238cb \
-                    size    64039
+checksums           rmd160  4bfb06b0ce790d9ed8f7cffafa34fedc278b31e2 \
+                    sha256  3847e205b19469f16ded05bda24b4758056d67ade4075a5ded4ce6628a9bad01 \
+                    size    47855
 
-if {${name} ne ${subport}} {
```

Comment:
this change doesn't look right - did you remove this `if`-block intentionally (and if so, why)?